AFQ/tests/test_api.py

```diff
@@ -349,7 +343,8 @@ def test_AFQ_data_waypoint():
     tgram = load_tractogram(myafq.bundles[0], myafq.dwi_img[0])
 
     bundles = aus.tgram_to_bundles(tgram, myafq.bundle_dict, myafq.dwi_img[0])
-    npt.assert_(len(bundles['CST_R']) > 0)
+    print(bundles)
```

Comment:
I think we'll want to remove this print statement before merging. 

Other than that - I think this is ready to go?

Reply:
Oh - nice catch! must have left that in there from debugging.